lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -95,23 +99,39 @@ export default function ContentSelector({
   };
 
   /** @param {File} file */
-  const selectLMSFile = file => {
+  const selectCanvasSFile = file => {
     cancelDialog();
     onSelectContent({ type: 'file', file });
   };
 
+  /** @param {File} file */
+  const selectBlackBoardFile = file => {
```

Comment:
```suggestion
  const selectBlackboardFile = file => {
```

For consistency with the rest of the code and Blackboard's product documentation.